### DISCUSSION TITLE
Import Users

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#240] [Q&A]
What is the best way to import a slew of new site users? I don't see an import option so I could jam them into mysql if there was a format, but how would the UserID get generated?

Any thoughts would be great... have about 500 records that need to be imported. I would rather not do them by hand.

**Comment by guustnieuwenhuis** [comment]
As far as I know, there is no bulk import option in the core of Masa CMS.

When tasked with this, I would write a script that uses the Masa CMS core to create these users. That script could be in a plugin, but a quicker/dirtier version could even be a template I would execute once (and then remove).
